module/VuFind/src/VuFind/Log/Handler/StreamHandler.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+/**
+ * Stream handler
+ *
+ * PHP version 8
+ *
+ * Copyright (C) Villanova University 2025.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ *
+ * @category VuFind
+ * @package  Error_Logging
+ * @author   Chris Hallberg <challber@villanova.edu>
+ * @author   Sambhav Pokharel <sambhav.pokharel@gmail.com>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org Main Site
+ */
+
+namespace VuFind\Log\Handler;
+
+use Monolog\Formatter\LineFormatter;
+use Monolog\Handler\StreamHandler as MonologStreamHandler;
+use Monolog\LogRecord;
+
+/**
+ * This class extends Monolog's StreamHandler to add verbosity control over log records
+ *
+ * @category VuFind
+ * @package  Error_Logging
+ * @author   Chris Hallberg <challber@villanova.edu>
+ * @author   Sambhav Pokharel <sambhav.pokharel@gmail.com>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org Main Site
+ */
+class StreamHandler extends MonologStreamHandler
+{
+    use VerbosityTrait;
+
+    protected ?LineFormatter $standardFileFormatter = null;
```

Comment:
This should have a standard comment on it:

```suggestion
    /**
     * Standard LineFormatter for file logging (used/initialized by getStandardFileFormatter())
     *
     * @var ?LineFormatter
     */
    protected ?LineFormatter $standardFileFormatter = null;
```